alfresco-actuators/src/main/java/eu/xenit/actuators/services/StatusInfoService.java

```diff
@@ -0,0 +1,83 @@
+package eu.xenit.actuators.services;
+
+import static eu.xenit.actuators.Health.KEY_ERROR;
+import static eu.xenit.actuators.Health.KEY_OUTPUT;
+
+import eu.xenit.actuators.Health;
+import eu.xenit.actuators.HealthIndicator;
+import eu.xenit.actuators.HealthStatus;
+import eu.xenit.actuators.exceptions.HealthException;
+import eu.xenit.actuators.model.gen.StatusInfo;
+import java.util.Collections;
+import java.util.Properties;
+import org.alfresco.repo.security.authentication.AuthenticationUtil;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.cmr.admin.RepoAdminService;
+import org.alfresco.service.cmr.audit.AuditService;
+import org.alfresco.service.cmr.thumbnail.ThumbnailService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+@Service
+public class StatusInfoService implements HealthIndicator {
+
+    private static final Logger logger = LoggerFactory.getLogger(StatusInfoService.class);
+
+    public static final String GLOBAL_HEALTH_ON_READONLY = "eu.xenit.actuators.repository.healthyOnReadOnly";
```

Comment:
I'm not sure I agree with using a config flag here. The reason you have this flag, is that you don't want your service getting kicked out of the load balancer, because it's going in read-only ?

Design goal: Organize health indicators into groups that can be used for different purposes. The load balancer can track one healthgroup, the orchestrator a different one.

(Health-group support should probably be a different PR, but still - if you include that flag now, someone is going to get surprised and upset, if it's removed in the future)

References:
* https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints.health.groups
* https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.spring-application.application-availability